HACKING.md

```diff
@@ -131,3 +131,47 @@ oc scale deployment machine-config-operator --replicas=0
    ```sh
    oc logs -f machine-config-daemon-<hash>
    ```
+   
+## Tests
+Tests can be executed on a per package basis with `go test` like so:
+ 
+`go test -v github.com/openshift/machine-config-operator/pkg/apis`
+
+All tests can be executed with:
+
+`make test`
+
+## Building Images
```

Comment:
How about moving the `Tests` and `Building Images` sections to the top since they're common to all the components? And then we can get into MCD-specific stuff.

Reply:
Done.